frappe/search/website_search.py

```diff
@@ -44,9 +44,8 @@ def get_items_to_index(self):
 			update_progress_bar("Retrieving Routes", i, len(routes))
 			self._items_to_index += [self.get_document_to_index(route)]
 
-		print()
 
-		return self.get_items_to_index()
+		return self._items_to_index
```

Comment:
Good, although this would still mean the subsequent call to get_items_to_index would recalculate this branch.